### PR TITLE
Feature/app actions

### PR DESCRIPTION
**Pull request by charanpreet-s**
1. Added ID in response
2. Changed type of timestamp variable

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -1874,6 +1877,26 @@ model assessment_cycles {
...
+  domain      String        @db.VarChar
+  action      String
+  app_actions app_actions[]
+}
+
+model app_actions {
+  id                 Int             @id @default(autoincrement())
+  android_actions_id Int
+  mentor_id          Int
+  actor_id           Int
```
Comment (karntrehan) [inline prisma/schema.prisma]: Why is this needed here? Are we keeping actions at each user level? Or actor level?

File: src/app.service.ts [hunk 2]
```diff
@@ -386,6 +386,40 @@ export class AppService {
     }
   }
 
+  async getAppActionsForMentor(
+    mentor: Mentor
```
Comment (karntrehan) [inline src/app.service.ts]: We should keep it actor specific. Actor ID can be sent here.

**Review by karntrehan — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (charanpreet-s) [reply prisma/schema.prisma]: Handling it for both actor and mentor level, if the field faces a production issue that is user specific, it can be resolved at the mentor level as well. The rest of the scenarios can be handled at the actor level.
For now, the domain is db, later it can be the ODK, clearing app data, and many more possibilities,

File: src/app.service.ts (hunk 2 above)
Comment (charanpreet-s) [reply src/app.service.ts]: If working with only actor id, the scope remains limited, thought to make it extensible to user level as well.
